lib/widget/switch_card.dart

```diff
@@ -0,0 +1,113 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+import '../hard_data.dart';
+import '../strings.dart';
+import '../style.dart';
+
+class SwitchCard extends StatefulWidget {
```

Comment:
I'm assuming this is intended to get wired up to the settings? If so once you do that if you expose the Settings via a Provider, this can become a stateless widget and the `_selected` state bool won't be needed.

Reply:
Yep planning that as per the PR Architecture changes comment. Still WIP.

Reply:
Stateless widget now using Provider for app  state, but still have Settings as a separate class for read/write to storage.